src/7-to-8/major-changes/scheduling.rst

```diff
@@ -23,10 +23,12 @@ Cylc can manage infinite workflows of repeating tasks:
 
 Cylc 8 has a new scheduling algorithm that:
 
-- Is much more efficient because it only has to manage active tasks
+- Is much more efficient because it doesn't need to track nearly as 
+  many waiting and succeeded tasks in addition to the active
+  (submitted and running) ones. 
 
   - waiting tasks are not pre-spawned before they are needed
-  - succeeded tasks are not kept across the active task window
+  - succeeded tasks are not retained across the :term:`active window`
```

Comment:
niff-naff style point - are these bullets sentences?

```suggestion
  - tasks the :term:`active window` when they succeed
```

Reply:
I can never quite decide if short bullet points should be styled as full sentences or not. But I'll rephrase along the lines of your suggestion (can't take it as-is as it seem malformed!).

Reply:
> I can never quite decide if short bullet points should be styled as full sentences or not. 

Don't hold me to consistency on this!